contracts/Market.sol

```diff
@@ -196,7 +198,7 @@ contract Market is IMarket, Ownable, ERC721 {
 			return 1;
 		}
 
-		pool -= _potentialPayout[propositionId]; //TODO: Should be _totalExposure;
+		pool -= _potentialPayout[propositionId];
```

Comment:
previously left comment said this should be `_totalExposure`, we don't want to do that anymore?

Reply:
correct